### PR TITLE
#31 add wind direction component

### PR DESCRIPTION
**Pull request by mrpeterhanley**
Takes in wind direction degrees (number) as a value since I saw that was the info provided from OpenWeatherMap API
Outputs the wind direction text (e.g. 'N', 'NNE' etc.) also rotates the icon according to the wind direction degrees

The component is used like this:
`<WindDirection degrees={180} />`

Untitled

### CODE REVIEW

**Review by inovux — CHANGES_REQUESTED** [review]
Looks good to me. Just want to mention one thing, could you use `rem` instead of `px` so we keep the codebase inline :) It's basically just `0.1 * px = rem`

**Comment by mrpeterhanley** [comment]
Thanks, updated values from px to rem (also for my previous components - IconButton, WeatherTitle

**Review by inovux — APPROVED** [review]

**Comment by inovux** [comment]
@hananina @mrpeterhanley Just solved the merge conflict, should be ready to merge now when Hana approves

File: src/components/WindDirection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react'
+import NavigationIcon from '@material-ui/icons/Navigation'
+import styles from './WindDirection.module.css'
+
+interface IWindDirection {
+  degrees: number
+}
+
+// http://snowfence.umn.edu/Components/winddirectionanddegrees.htm
+const windDirectionValues = {
```
Comment (hananina) [inline src/components/WindDirection/index.tsx:10]: 📝👀

**Review by hananina — APPROVED** [review]
Look good to me🎊🎊

**Pull request merged**